props/saber_fett263_buttons.h

```diff
@@ -2488,18 +2459,348 @@ SaberFett263Buttons() : PropBase() {}
       LengthPreview(blade_num_);
       menu_type_ = MENU_LENGTH;
       break;
-    case MENU_COPY_COLOR:
-      if (choice_) {
-        current_preset_.SetStyle(copy_blade_, style_parser.CopyArguments(current_preset_.GetStyle(blade_num_), current_preset_.GetStyle(copy_blade_), int_arg_menu_, NELEM(int_arg_menu_)));
-        DOVALIDATE(current_preset_);
-        current_preset_.Save();
-        DOVALIDATE(current_preset_);
-        show_preview_.Stop(blade_preview_);
-        UpdateStyle();
-        menu_type_ = MENU_COLOR_SUB;
-        MenuSelect();
-        choice_ = false;
-        blade_preview_ = 0;
+#endif
```

Comment:
Duplicating all this code is bad.
We need to find a way to do this that doesn't cause so much code duplication.

Reply:
This isn't duplicated, I moved it from other locations in the same switch to group together.  You'll see corresponding code removed from the lines lower down.

Reply:
Sorry about that, it's really hard to tell what's going on with this change since there is so much stuff moving around. Smaller changes would have been nice.